### PR TITLE
feat: start MinIO pools in --json/--anonymous format

### PR DESCRIPTION
**Pull request by harshavardhana**
optionally allow them to be disabled

### CODE REVIEW

**Review by nitisht — APPROVED** [review]
LGTM

File: pkg/apis/minio.min.io/v2/types.go [hunk 1]
```diff
@@ -284,6 +269,33 @@ type TenantSpec struct {
...
+	// * `CONSOLE_SECRET_KEY` - The "Password" for the MinIO user +
+	//
+	// The Operator creates each user with the `consoleAdmin` policy by default. You can change the assigned policy after the Tenant starts. +
+	// +optional
+	Users []*corev1.LocalObjectReference `json:"users,omitempty"`
+	// *Optional* +
+	//
+	// Enable JSON, Anonymous logging for MinIO tenants.
+	// +optional
+	Logging *Logging `json:"logging,omitempty"`
```
Comment (dvaldivia) [inline pkg/apis/minio.min.io/v2/types.go:291]: We need to add this on the `types.go` for `v1` as well

Comment (dvaldivia) [reply pkg/apis/minio.min.io/v2/types.go:291]: Also in `pkg/apis/minio.min.io/v1/conversion.go` you should add support to copy this new field from v1 to v2 and viceversa

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: pkg/apis/minio.min.io/v2/types.go (hunk 1 above)
Comment (harshavardhana) [reply pkg/apis/minio.min.io/v2/types.go:291]: why v1? @dvaldivia

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**